.github/workflows/job_build_mlir_linux.yml

```
@@ -0,0 +1,221 @@
+name: Linux MLIR Build
+on:
+  workflow_call:
+    inputs:
+      os:
+        description: 'OS that is used for building in the form of "ubuntu_20_04"'
+        type: string
+        required: true
+      build-runner:
+        description: 'Machine on which the builds will run'
+        type: string
+        required: true
+      openvino-organization:
+        description: 'OpenVINO fork owner to clone'
+        type: string
+        required: true
+      openvino-commit:
+        description: 'OpenVINO commit to checkout'
+        type: string
+        required: true
+      openvino-cmake-options:
+        description: 'A string of options passed to OpenVINO CMake'
+        type: string
+        required: false
+      npu-cmake-options:
+        description: 'A string of options passed to NPU Compiler CMake'
+        type: string
+        required: false
+      build-cache:
+        description: 'Whether to take previously built OpenVINO/NPU package.'
```

Nitpick, for description consistency
```suggestion
        description: 'Whether to take previously built OpenVINO/NPU package'
```